cmd/run.go

```diff
@@ -37,6 +40,10 @@ func init() {
 	runCmd.PersistentFlags().StringArrayVarP(&taskRequest.Mounts, "mount", "m", []string{}, "bind-mount a volume the task")
 	runCmd.PersistentFlags().StringArrayVarP(&taskRequest.Secrets, "secret", "s", []string{}, "Add existing secret to task")
 
+	runCmd.PersistentFlags().StringVarP(&taskRequest.ReportUrl, "report-url", "p", "", "Report URL for Helio")
+	runCmd.PersistentFlags().StringVarP(&taskRequest.HeartbeatUrl, "heartbeat-url", "y", "", "Heartbeat URL for Helio")
+	runCmd.PersistentFlags().StringVarP(&taskRequest.JobToken, "job-token", "j", "", "Job Token for Helio API access")
```

Comment:
I'd prefer if this is somehow made more generic so we could get it merged upstream, in case upstream is still maintained.
Should we ask if they are interested in such a feature? 

How to make it more generic is a bit tricky though. I imagine that you can pass a  "Heartbeater" and a "Reporter" type (interface type with each a method `Heartbeat` / `Report`) to runTask.
Then we'd create an implementation which fulfills those methods and executes an HTTP request to a URL with specified method and headers. 
What we'd need then is a flag which says: I want a heartbeater of this type and that type gets these parameters (for the HTTP request implementation: method, URL, headers (which includes the job token in our case)). 

Something for later, I guess. But I'll think about it.